src/Core/Repository.ts

```diff
@@ -0,0 +1,38 @@
+import { Model } from "mongoose";
```

Comment:
move this file from core package to repository package

Reply:
done